### PR TITLE
[DAT-1269] - Redirect to the login when the user is cancelated

### PR DESCRIPTION
**Pull request by jhoffman-ms**
Redirect to the login when the user is cancelated

**Task:** [DAT-1269](https://makingsense.atlassian.net/jira/software/projects/DAT/boards/62?selectedIssue=DAT-1269)

### CODE REVIEW

**Comment by ms-fcabrera** [comment]
👍

File: src/components/Plans/Checkout/PaymentMethod/PaymentMethod.js [hunk 1]
```diff
@@ -374,7 +375,11 @@ export const PaymentMethod = InjectAppServices(
         setError({ error: false, message: '' });
         handleSaveAndContinue();
       } else {
-        setError({ error: true, message: handleMessage(result.error) });
+        if (result.error.response.data === userCanceledError) {
+          window.location.href = '/login';
```
[the remark this answers is not in the document]
Comment (jhoffman-ms) [reply src/components/Plans/Checkout/PaymentMethod/PaymentMethod.js:379]: Ok, let me check.

[the remark this answers is not in the document]
Comment (jhoffman-ms) [reply src/components/Plans/Checkout/PaymentMethod/PaymentMethod.js:379]: Done

**Comment by jcamposmk** [comment]
👍

**Pull request merged**